### PR TITLE
feat: Rename from sugar to core module name

### PR DESCRIPTION
**Pull request by esloch**
## How to test these changes

* ```...```

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [x] new feature
- [x] maintenance

About this PR:
- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:
- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more complexity.
- [ ] New and old tests passed locally.

## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved .
```

### CODE REVIEW

File: src/sugar/cli.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from pathlib import Path
 from typing import Tuple
 
-from sugar import Sugar
+from core import Sugar
```
Comment (xmnlab) [inline src/sugar/cli.py:9]:
```suggestion
from sugar.core import Sugar
```

File: src/sugar/core.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 from jinja2 import Template
 
-from sugar import __version__
+from core import __version__
```
Comment (xmnlab) [inline src/sugar/core.py:16]:
```suggestion
from sugar import __version__
```

File: tests/test_containers_sugar.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-from sugar import Sugar
+from core import Sugar
```
Comment (xmnlab) [inline tests/test_containers_sugar.py:7]:
```suggestion
from sugar.core import Sugar
```

**Review by xmnlab — CHANGES_REQUESTED** [review]

**Comment by xmnlab** [comment]
@esloch I just merged this PR: https://github.com/osl-incubator/sugar/pull/102
could you rebase your branch on top of the upstream main?

**Comment by esloch** [comment]
> @esloch I just merged this PR: #102 could you rebase your branch on top of the upstream main?

I think that we need to update the poetry.lock.

**Comment by esloch** [comment]
> Could you remove and recreate it please?

I removed the "poetry.lock" and then run the "poetry update":

``` 
(sugar) ✔ /workspace/repos/osl/sugar [rename-module|● 1] 
21:39 $ poetry update
Updating dependencies
Resolving dependencies... (65.4s)

The current project's Python requirement (>=3.8.1) is not compatible with some of the required packages Python requirement:
  - makim requires Python >=3.8.1,<4.0.0, so it will not be satisfied for Python >=4.0.0

Because containers-sugar depends on makim (1.8.3) which requires Python >=3.8.1,<4.0.0, version solving failed.

  • Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties
    
    For makim, a possible solution would be to set the `python` property to ">=3.8.1,<4.0.0"
   ```

**Comment by xmnlab** [comment]
@esloch the issue with the installation was fixed (I hope!)
now the error is about the tests.

**Comment by esloch** [comment]
> @esloch the issue with the installation was fixed (I hope!) now the error is about the tests.

Thanks Ivan!

**Comment by xmnlab** [comment]
@esloch , just cut a new release. please, rebase your branch on top of the upstream main

**Pull request merged**